templates/etc/systemd/system/sftp-storage-cleanup.timer

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Regularly cleans out the temporary storage used by the sftp service
```

Comment:
I opted to use systemd instead of cron just because:

1. we're already using systemd and it felt a little bit more explicit.
2. It now becomes possible to manually run the cleanup service `systemctl start sftp-storage-cleanup`
3. The service can use the shared environment file (e.g. point to the same temporary storage directory)